src/flatland/ordered/map.clj

```diff
@@ -133,7 +123,7 @@
     (APersistentMap/mapHash this))
   IHashEq
   (hasheq [this]
-    (hasheq-ordered-map this))
+    (hash-unordered-coll this))
```

Comment:
Also, how is this related to Clojure < 1.6 ?

Reply:
See other comment.